lib/Text/SExpression/Internal.hs

```diff
@@ -77,15 +81,17 @@ symbol = oneOf "!$%&|*+-/:<=>?@^_~#"
 
 -- | S-expression parser
 parseSExpr ::
+    LiteralParsers ->
```

Comment:
This API change breaks the code sample at https://github.com/rcook/sexpr-parser/blob/master/lib/Text/SExpression.hs.